write_bc_fluent.py

```diff
@@ -0,0 +1,46 @@
+import ansys.fluent.core as pyfluent
+import sys
+from pathlib import Path
+from contextlib import redirect_stdout
+import numpy as np
+import multiprocessing
+
+# Read input from bash-script
+if len(sys.argv) != 6:
+    sys.exit("5 arguments should be used: dimensions - case path - case name - start time step - inlet boundary name.")
+dimensions = int(sys.argv[1])
+case_path = Path(sys.argv[2])
+case_name = str(sys.argv[3])
+time_step_start = int(sys.argv[4])
+boundary_name = str(sys.argv[5])
+
+VOFw = np.load('inletDefinition-VOFw.npy')
+face_ids = np.load('face_ids.npy')
+np.savetxt(f'inlet_VOFw_start{time_step_start}.dat', VOFw.squeeze(), fmt='%i')
+np.savetxt('face_ids.dat', face_ids, fmt='%i', header=f'{face_ids.shape[0]}')
+
+with open('write_bc.log', 'w') as logfile:
+    with redirect_stdout(logfile):
+        # Launch fluent and read case
+        session = pyfluent.launch_fluent(mode="solver", precision="double", version=f"{dimensions}d",
+                                         processor_count=multiprocessing.cpu_count(), show_gui=True,
+                                         cleanup_on_exit=False)
```

Comment:
Why no cleanup_on_exit?

Reply:
I am currently using the code in an interactive mode, so I don't want Fluent to shut down. Later, I will try to streamline it for a batch mode.